tests/unit/libexpr/derivation.cc

```diff
@@ -0,0 +1,46 @@
+#include "gtest/gtest.h"
```

Comment:
This test can probably be done with less code in a functional test, i.e. something like
```
drvPath=$(nix-instantiate foo.nix)
cmp $(drvPath) $TEST_ROOT/advanced-attributes.drv
```

Reply:
Let me know what you think about the new approach (inspired by `tests/functional/lang`) :D